src/Util/AdminObjectAclData.php

```diff
@@ -160,11 +144,11 @@ public function setAcl(MutableAclInterface $acl)
 
     /**
      * Gets ACL.
-     *
-     * @return MutableAclInterface
      */
-    public function getAcl()
+    public function getAcl(): MutableAclInterface
     {
+        \assert(null !== $this->acl);
```

Comment:
We can't assert that `this->acl` is not null.

So I would update the return type to `?MutableAclInterface`